packages/wallet/src/redux/protocols/direct-funding/reducer.ts

```diff
@@ -159,13 +159,19 @@ const commitmentReceivedReducer: DFReducer = (
       sharedData: sharedDataWithOwnCommitment,
     };
   } else {
+    const checkResult = checkAndStore(sharedData, action.signedCommitment);
```

Comment:
Looks like `checkAndStore` is performed (and the result is inspected) for all player B code paths. We can probably factor that logic outside of the player B `if/else` (in other words, to the top of the player B codepath).